app/toolkit/sevtool.py

```diff
@@ -84,4 +84,4 @@ def get_platform_status(self) -> SevPlatformStatus:
         return SevPlatformStatus.from_dict(sev_platform_status)
 
     def export_certificates(self):
-        _ = self.sevtool_cmd("--export_cert_chain", output_dir=self.certificates_dir)
+        _ = self.sevtool_cmd("export", "--full", self.certificates_archive)
```

Comment:
This shouldn't be full to work
```suggestion
        _ = self.sevtool_cmd("export", self.certificates_archive)
```